quimb/evo.py

```diff
@@ -336,7 +345,12 @@ def ham(t):
                     make_immutable(Ht)
                 return Ht
 
-        self._setup_callback(compute)
+        if (int_stop is not None) and (not method == 'integrate'):
```

Comment:
I think ``method != 'integrate'`` is a bit more readable.

Reply:
Agreed.